flocx_market/tests/unit/test_matcher.py

```diff
@@ -0,0 +1,130 @@
+from unittest import TestCase
```

Comment:
You don't need this import (using pytest, we just write methods, rather than classses).